pkgs/development/compilers/elm/default.nix

```diff
@@ -217,6 +217,19 @@ in lib.makeScope pkgs.newScope (self: with self; {
         };
       };
 
+      elm-spa = nodePkgs."elm-spa".overrideAttrs  (
+        old: {
+          nativeBuildInputs = (old.nativeBuildInputs or []) ++ [ makeWrapper old.nodejs.pkgs.node-gyp-build ];
+
+          meta = with lib; nodePkgs."elm-spa".meta // {
+            description = "A tool for building single page apps in Elm";
+            homepage = "https://www.elm-spa.dev/";
+            license = licenses.bsd3;
+            maintainers = [ maintainers.ilyakooo0 ];
```

Comment:
Let me know if you would like me to add myself there as well. But I'm more than happy to let you maintain this package yourself.

Reply:
I would gladly add you as a maintainer (:

Reply:
Oh, you merged it already

Reply:
yeah no worries I can add myself in some next PRs. It's probably even more systematic that way - users adding themselves rather than letting others to do so.